README.md

```diff
@@ -38,8 +38,8 @@ Visit [flux.paradigm.xyz](https://flux.paradigm.xyz) to try Flux online or follo
 
 ```sh
 git clone https://github.com/transmissions11/flux.git
-npm install
-npm run dev
+pnpm install
+pnpm run dev
```

Comment:
i'm sorry to do this to you but i've changed my mind on the pnpm thing, i'd prefer to stick with npm to make it easier for contributors of all shapes and sizes. let's revert here please

Reply:
Ok no problem, will revert.

Reply:
ty ser sorry for the trouble 🙏

Reply:
no worries 🫡